pkg/webui/components/table/table/index.js

```diff
@@ -38,16 +38,21 @@ Empty.defaultProps = {
   message: undefined,
 }
 
-const Head = ({ className, ...props }) => (
-  <div {...props} className={classnames(className, style.sectionHeader)} />
+const Head = ({ className, panelStyle, ...props }) => (
+  <>
+    {!panelStyle && <div className={style.sectionDivider} />}
+    <div {...props} className={classnames(className, style.sectionHeader)} />
+  </>
```

Comment:
```suggestion
const Head = ({ className, panelStyle, children, ...props }) => (
  <>
    <div {...props} className={classnames(className, style.sectionHeader)}>
      {!panelStyle && <div className={style.sectionDivider} />}
      {children}
    </div>
  </>
```

See my other comment.